### PR TITLE
Awesome books: plain JavaScript with objects Feature

### PR DESCRIPTION
**Pull request by disc3110**
- created an array with books 
- created functions to add and remove books
- displayed all books including the new ones 
- saved data using local storage

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    author,
+  };
+
+  books.push(newBook);
+  setStorage();
+  renderBooks(books);// eslint-disable-line
+}
+
+function removeBook(id) {
+  books.splice(id, 1);
```
Comment (Davidosky007) [inline script.js]: * This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* Using the array method `splice` might be expensive as it mutates the original array which may lead to unexpected and hard-to-debug issues [Read more on why it is not advisable](https://blog.sapegin.me/all/avoid-mutation/)

File: script.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  }
+}
+
+function addBook(title, author) {
+  const newBook = {
+    title,
+    author,
+  };
+
+  books.push(newBook);
```
Comment (Davidosky007) [inline script.js]: * This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* This method `push` also mutates the array of books you could use other array methods that returns a new array instead.

**Review by Davidosky007 — APPROVED** [review]
Hi @disc3110 @brenoxav, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
doing-good-gif

# [OPTIONAL]
* I left you some OPTIONAL comments i strongly suggest you look into them in your free time as you would learn more ways to properly mutate array objects 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by brenoxav** [comment]
### Awesome books: refactor to use JavaScript classes

- Refactor code to use Classes whenever possible
- Created methods for adding and removing books
- All the original functionality maintained.

**To the Code Reviewer:**
We accidentally merged the milestone 2 commits to the old PR and couldn't submit a new PR.
Please send instructions on how to proceed.

**Review by voscarmv — COMMENTED** [review]
Hi @brenoxav , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted an invalid Pull Request. If you're unable to submit a different PR to the form, please at least provide a link to the correct one in the comments here next time. All I see is this message:

image

Which references some "instructions", but I see no instructions anywhere. Please fix this by submitting the correct pull request on the form, or providing a link here.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by voscarmv** [comment]
@brenoxav the pull request is supposed to show proper git flow. That is: it's supposed to not be merged into main yet. It's supposed to show me linter messages. It's supposed to have a particular description about what was learned in creating it. You need all of these things to get a proper code review. Please make sure to do all these things before submitting again next time!

Feel free to open a completely new Pull Request and just reference this one there, to provide context to the new code reviewer.

Cheers!

**Pull request merged**